source/extensions/filters/http/common/jwks_fetcher.cc

```diff
@@ -0,0 +1,91 @@
+#include "extensions/filters/http/common/jwks_fetcher.h"
+
+#include "common/common/enum_to_int.h"
+#include "common/http/headers.h"
+#include "common/http/utility.h"
+
+#include "jwt_verify_lib/status.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Common {
+namespace {
+class JwksFetcherImpl : public JwksFetcher,
```

Comment:
Nit: prefer blank line between start of namespace and class definiton.

Reply:
Fixed in 8fa1dc8